### PR TITLE
Rename Nitrogen-Fixing Plastid to Nitroplast

### PR DESCRIPTION
**Pull request by HexapodPhilosopher**
**Brief Description of What This PR Does**

This PR does some stuff...

**Related Issues**

**Progress Checklist**

Note: before starting this checklist the PR should be marked as non-draft.

- [X] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [x] Initial code review passed (this and further items should not be checked by the PR author)
- [x] Functionality is confirmed working by another person (see above checklist link)
- [x] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author. Merging must follow our
[styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md#git).

### CODE REVIEW

**Comment by GameDungeon** [comment]
https://suggestions.revolutionarygamesstudio.com/posts/517/rename-nitrogen-fixing-plastid-into-nitroplast

File: locale/en.po [hunk 1]
```diff
@@ -7268,6 +7238,36 @@ msgstr ""
 msgid "WIKI_NITROGENASE_UPGRADES"
 msgstr "No upgrades."
 
+msgid "WIKI_NITROPLAST_EFFECTS"
```
Comment (hhyyrylainen) [inline locale/en.po:7241]: There's a big problem here, the wiki content is automatically updated with a script from our wiki, and I don't see these edits on the wiki (at least there isn't a page called "nitroplast").

So could you make these edits on the wiki? I'll help out by moving the existing page to be named nitroplast you can then edit: https://wiki.revolutionarygamesstudio.com/wiki/Nitroplast

**Review by hhyyrylainen — CHANGES_REQUESTED** [review]

File: locale/en.po (hunk 1 above)
Comment (HexapodPhilosopher) [reply locale/en.po:7241]: Do these items only need to be updated on the wiki, or should the changes be done both here and on the wiki?

Comment (HexapodPhilosopher) [reply locale/en.po:7249]: It didn't seem to make any changes to my local files - at least nothing showed in VSCode.

Comment (hhyyrylainen) [reply locale/en.po:7249]: Has the wiki website been actually updated now?

Comment (HexapodPhilosopher) [reply locale/en.po:7249]: Yes, I updated the wiki page for Nitroplast.

Comment (hhyyrylainen) [reply locale/en.po:7249]: It looks like the script has now worked as the changes look correct to me now.

**Review by hhyyrylainen — APPROVED** [review]
I think you *didn't* use the update script as I just tried running it and it found multiple problems in the wiki text so I went to the wiki and fixed those, and made the script itself a bit better at giving errors. So now I think this should be fine and the text is up to date.

**Pull request merged**